test/unit/wiki_extensions_child_pages_count_test.rb

```diff
@@ -4,19 +4,40 @@ class WikiExtensionsChildPagesCountTest < Redmine::HelperTest
   include ApplicationHelper
   include ERB::Util
 
-  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues
+  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues,
+           :roles, :enabled_modules, :users
+
+  def setup
+    super
+    @project = nil
```

Comment:
ここで初期化してしまうのがよさそうに思います。

```suggestion
    @project = Project.find(1)
```

Reply:
fix: [88c7d2d](https://github.com/otegami/redmine_wiki_extensions/pull/5/commits/88c7d2df0b383c4564dba53df544a56726674f3f) テストケースの前提と必要なことがわかりやすいように移動する修正を行いました。